lib/Runtime/Language/InlineCache.inl

```diff
@@ -93,7 +93,15 @@ namespace Js
             Assert(propertyObject->GetScriptContext() == requestContext); // we never cache a type from another script context
             Assert(u.accessor.flags & InlineCacheGetterFlag);
 
-            RecyclableObject *const function = RecyclableObject::UnsafeFromVar(u.accessor.object->GetInlineSlot(u.accessor.slotIndex));
+            RecyclableObject * function;
+            if (u.accessor.isOnProto)
+            {
+                function = RecyclableObject::UnsafeFromVar(u.accessor.object->GetInlineSlot(u.accessor.slotIndex));
+            }
+            else
+            {
+                function = RecyclableObject::UnsafeFromVar(DynamicObject::FromVar(propertyObject)->GetInlineSlot(u.accessor.slotIndex));
```

Comment:
>FromVar [](start = 74, length = 7)

I think you can use UnsafeFromVar here since 'type' is the same as the cached type.

Reply:
Cool. Thanks.